EventFilter/RawDataCollector/src/RawDataCollectorByLabel.cc

```diff
@@ -71,10 +71,10 @@ void RawDataCollectorByLabel::produce(Event & e, const EventSetup& c){
        if(verbose_ > 1) std::cout << "Copying data from FED #" << j << std::endl;
        FEDRawData & fedDataProd = producedData->FEDData(j);
        if ( fedDataProd.size() != 0 ) {
-	 if(verbose_ > 1) {
-	   std::cout << " More than one FEDRawDataCollection with data in FED ";
-	   std::cout << j << " Skipping the 2nd\n";
-	 }
+        if(verbose_ > 1) {
```

Comment:
yes, my mistake, I opened the file and for readability in my editor I modified the indentation.  While committing I didn't realized that it entered in the PR as well also if it was not supposed to be. I marked it as resolved because it was just an indentation issue that should be done.

Reply:
Note, however, is that the indentation is now inconsistent with the rest of the file, which elevates the chances of someone else having to revert your change later.